### PR TITLE
add keb version to issue body

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

https://github.com/thoth-station/kebechet/issues/888

## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

File: kebechet/utils.py [hunk 1]
```diff
@@ -177,8 +178,8 @@ def get_issue_by_title(ogr_project: BaseGitProject, title: str):
 
 
 EXCEPTION_ISSUE_BODY = """## Description
-This is an automated issue generated by Kebechet. The {manager} manager threw an exception ({exception_type}) at
-runtime while working on the following repository https://github.com/{slug}
+This is an automated issue generated by Kebechet v{version}. The {manager} manager threw an exception ({exception_type})
```
Comment (codificat) [inline kebechet/utils.py:181]: It's imported as `keb_version`, right?
```suggestion
This is an automated issue generated by Kebechet v{keb_version}. The {manager} manager threw an exception ({exception_type})
```

Comment (KPostOffice) [reply kebechet/utils.py:181]: I called it below by writing:
```
EXCEPTION_ISSUE_BODY.format(..., version=keb_version,...)
```

Comment (codificat) [reply kebechet/utils.py:181]: ah, right, sorry I missed that :+1:

**Pull request merged**